packages/wdio-config/src/lib/FileSystemPathService.ts

```diff
@@ -36,12 +36,18 @@ export default class FileSystemPathService implements PathService {
          * and we also want to be able to find files with these characters included
          * we add an additional check to see if the file as pattern exists.
          * add file to globResult only if filename doesn't include pattern(*)
-         * and globResult doest contain the fileName
+         * and globResult doesn't contain the fileName
          * and file should be available
          */
         if (!pattern.includes('*') && !globResult.includes(pattern) && !globResult.includes(fileName) && fs.existsSync(fileName)) {
             globResult.push(fileName)
         }
+
+        // alphabetically sort spec files
+        if (Array.isArray(globResult) && globResult.length) {
+            globResult.sort()
+        }
```

Comment:
I don't think we need to verify if `globResult` is an array. It should based on it's initiation.
```suggestion
        globResult.sort()
```

Reply:
That was the original implementation, however, it was causing a unit test to fail - https://github.com/webdriverio/webdriverio/blob/main/packages/wdio-config/tests/FileSystemPathService.test.ts#L60

```TypeError: globResult.sort is not a function
 ❯ FileSystemPathService.glob packages/wdio-config/src/lib/FileSystemPathService.ts:51:27
     49|         // }
     50|
     51|         return globResult.sort()
       |                           ^
     52|     }
     53|
 ❯ packages/wdio-config/tests/FileSystemPathService.test.ts:62:24
 ```

Reply:
Adding `vi.mocked(glob.sync).mockReturnValue([])` before calling the method in the test will help you to establish the right return value of the glob function.